nodestream_plugin_neptune/neptune_connection.py

```diff
@@ -0,0 +1,100 @@
+import json
+from abc import ABC, abstractmethod
+from logging import getLogger
+
+from aiobotocore.session import get_session
+
+
+class NeptuneConnection(ABC):
+    @abstractmethod
+    async def execute(self, query_stmt: str, parameters: list):
+        pass
+
+
+class NeptuneDBConnection(NeptuneConnection):
+    @classmethod
+    def from_configuration(cls, host: str, region: str):
+        return cls(host=host, region=region)
+
+    def __init__(self, host: str, region: str) -> None:
+        self.host = host
+        self.region = region
+        self.session = get_session()
```

Comment:
Would this be better named as `boto_session`? Since we're talking about databases, I'd rather this not be confused with a transaction or connection to the database. I know thats not how neptune does it, but seems common enough of a concept it may cause confusion. Thoughts? 

```suggestion
        self.boto_session = get_session()
```